electron/webpack/webpack.config.main.ts

```diff
@@ -0,0 +1,69 @@
+import type { Configuration } from 'webpack';
+import type IForkTsCheckerWebpackPlugin from 'fork-ts-checker-webpack-plugin';
+import TerserPlugin from 'terser-webpack-plugin';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const ForkTsCheckerWebpackPlugin: typeof IForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+import paths from './paths';
```

Comment:
Seems like the linting isn't running on these files. I would have expected this import to be flagged.

Let's make sure that is set up and running on CI.

Reply:
This import is because of weirdness in the `fork-ts-checker-webpack-plugin` but will setup linting.